code/game/objects/items/manuals.dm

```diff
@@ -958,15 +958,17 @@
 	title = "Hacking"
 	page_link = "Hacking"
 
-///Reusable books that grant actions (knowledge is power)
-/obj/item/book/action_granting

```

Comment:
you didn't change the name of these typepaths in places like cargo

Reply:
fuck cargo i'm going off the grid (okay i'll fix it later thank you)

Reply:
done!